content/before-you-fly/code-of-conduct.md

```diff
@@ -6,46 +6,45 @@ weight: 2
 featured_image: "/img/become-a-member.jpg"
 ---
 
-The Blue Mountains Soaring Club thanks you for your membership contribution. We also welcome your ideas and the giving of your time to club meetings and projects. Your club supports your access to sites but needs your support in return.
+The Blue Mountains Soaring Club (BMSC) thanks you for your membership contribution. We also welcome your ideas and the giving of your time to club meetings and projects. Your club supports your access to sites but **needs your support in return**.
```

Comment:
Just added "(BMSC)" to be consistent with other acronym definitions on the page. Also bolded  "needs your support in return" (the double asterisk just tells the website publishing to make that text bold), as you can see on the staging website. You'll see other places below where I've bolded what I think is the important part. If you think differently, happy to sit down and do that together sometime.